R/SS_changepars.R

```diff
@@ -331,7 +331,7 @@ SS_changepars <-
     if (verbose) {
       message(
         "Wrote new file to ", newctlfile, " with the following changes:\n",
-        paste0(capture.output(results), collapse = "\n")
+        paste0(utils::capture.output(results), collapse = "\n")
```

Comment:
I like the practice of adding the package name to functions like this in general because it makes clear where the function is coming from and easier to figure out elements of the required packages are being used. However, in this case {utils} is part of the base R installation, so what does it add? Certainly there's no harm in adding this, I'm just curious.

Reply:
There was a warning about this in r cmd check, which is the only reason I added it!

Reply:
Makes sense. Thank you for making the warning go away.